JavaScript/Github_unfollowers/README.md

```diff
@@ -0,0 +1,18 @@
+# Github_Unfollowers
+This is a script to track recent unfollowers on github. 
```

Comment:
Replace this with `Github Unfollowers is a Script to track the recent accounts who unfollowed you on Github`